src/components/Kv/KvButton.vue

```diff
@@ -57,6 +57,10 @@ export default {
 	font-weight: $button-font-weight;
 	box-shadow: $button-primary-shadow;
 
+	&.rounded {
+		border-radius: rem-calc(10);
+	}
```

Comment:
@ryan-ludwig At some point in the near future we may want to collect the various "rounded" settings we have around Ui and try to establish a couple of prescribed options...